csharp/test/Ice/properties/Client.cs

```diff
@@ -96,20 +97,48 @@ public override void Run(string[] args)
 
                 using var communicator = new Communicator(timeSpanProperties);
 
-                var duration = communicator.GetPropertyAsTimeSpan("Duration.Milliseconds");
-                TestHelper.Assert(duration.Equals(TimeSpan.FromMilliseconds(10000)));
+                {
+                    var duration = communicator.GetPropertyAsTimeSpan("Duration.Milliseconds");
+                    TestHelper.Assert(duration.Equals(TimeSpan.FromMilliseconds(100)));
+                    TestHelper.Assert(duration.HasValue && duration.Value.ToPropertyString().Equals("100ms"));
```

Comment:
You should use `==` here and no need for `TestHelper.` as the client extends it is just a member method.
```
Assert(duration == TimeSpan.FromMilliseconds(100));
Assert(duration.Value.ToPropertyString() == "100ms");
```